token/cli/src/main.rs

```diff
@@ -100,7 +101,39 @@ pub const MULTISIG_SIGNER_ARG: ArgConstant<'static> = ArgConstant {
     help: "Member signer of a multisig account",
 };
 
-pub const CREATE_TOKEN: &str = "create-token";
+#[derive(Debug, Clone, Copy, PartialEq)]
+#[derive(Display, EnumString, IntoStaticStr)]
+#[strum(serialize_all = "kebab-case")]
+pub enum CommandName {
+    CreateToken,
+    Close,
+    Bench,
+    CreateAccount,
+    CreateMultisig,
+    Authorize,
+    Transfer,
+    Burn,
+    Mint,
+    Freeze,
+    Thaw,
+    Wrap,
+    Unwrap,
+    Approve,
+    Revoke,
+    Balance,
+    Supply,
+    Accounts,
+    Address,
+    AccountInfo,
+    MultisigInfo,
+    Gc,
+    SyncNative
+}
+
+pub fn to_str(command:CommandName) -> &'static str {
+    let command_str : &'static str = command.into();
+    command_str
+}
```

Comment:
Let's remove this function and just call `.into()` on the `CommandName` directly